docs/get_sql_db.md

```diff
@@ -0,0 +1,43 @@
+# get_sql_db
+
+""" Retrieves summary information for Microsoft SQL databases. Each keyword argument is a query parameter to filter the database details returned i.e. you can query for a specific database name, hostname, instance, is_relic, effective_sla_domain etc.
+```py
+def get_sql_db(instance_id=None, availability_group_id=None, effective_sla_domain_id=None, primary_cluster_id=None, name=None, instance=None, hostname=None, sla_assignment=None, limit=None, offset=None,  is_relic=None, is_live_mount=None, is_log_shipping_secondary=None, sort_by=None, sort_order=None, timeout=15)
+```
+
+## Keyword Arguments
+| Name                      | Type | Description                                                                 | Choices | Default |
+|---------------------------|------|-----------------------------------------------------------------------------|---------|---------|
+| name                      | str  | Filter by a substring of the database name.                                 |         |         |
+| instance                  | str  | The SQL instance name of the database.                                      |         |         |
+| hostname                  | str  | The SQL host name of the database.                                          |         |         |
+| instance_id               | str  | Filter by Microsoft SQL instance.                                           |         |         |
+| availability_group_id     | str  | Filter by the id of an Always On Availability Group.                        |         |         |
```

Comment:
Filter by the ID of an Always On Availability Group.

Reply:
`ID` should be captalized

Reply:
Completely missed that 👍